### PR TITLE
Change popup color in darkmode

### PR DESCRIPTION
**Pull request by spekss**
I think the color is too much blue and that's not make a good contrast

preview: 
 new
 new
 old

### CODE REVIEW

**Comment by AustinXII** [comment]
Idk If it's just my screen but I really don't see a difference

**Comment by spekss** [comment]
The old is more blue that the new

File: style/client.css [hunk 1]
```diff
@@ -3230,12 +3230,12 @@ a.ilink.yours {
...
 
 .dark .ps-popup {
-	background: #0D151E;
+	background: #1e2225;
 	color: #DDD;
 	border-color: #888;
 
 	box-shadow: 2px 2px 3px rgba(0,0,0,.2);
-}
+}" height
```
Comment (Zarel) [inline style/client.css]: ...

**Comment by Zarel** [comment]
The bigger problem might be to lighten link colors, if contrast is an issue.

**Comment by spekss** [comment]
oups sorry for that I had written it to display the screen, and I have missclick and write it here

**Comment by Zarel** [comment]
This doesn't seem to be necessary and the criticisms don't seem to be handled. Closing.

**Pull request closed without merging**